apps/remix-ide-e2e/src/tests/editor.test.ts

```diff
@@ -97,9 +100,11 @@ module.exports = {
     .click('li[key="browser/removeSourcehighlightScript.js"]')
     .pause(2000)
     .executeScript('remix.exeCurrent()')
-    .waitForElementVisible('li[key="browser/3_Ballot.sol"]')
-    .click('li[key="browser/3_Ballot.sol"]')
-    .pause(2000)
+    .waitForElementVisible('li[key="browser/contracts"]')
+    .click('li[key="browser/contracts"]') // files don't appear, so we click twice to get the files
```

Comment:
That sounds a bit weird, is this the only case this happens?

Reply:
Yes. I will open an issue for this.

Reply:
Here it is: https://github.com/ethereum/remix-project/issues/670

Reply:
this also happens here: https://github.com/ethereum/remix-project/pull/659/files#diff-be796db79c0c7616c194328f2833fc50a1cfb57fe105aaa4a390f89e30aa1513R46